### PR TITLE
DOCS-3292 Adds iOS Mobile Vitals Documentation

### PR DESCRIPTION
**Pull request by alai97**
### What and why?

Adds Mobile Vitals doc page in the iOS SDK repo + removes a Crash Reporting beta banner.

### How?

DOCS-3292

### Additional Notes

Preview link is available in Documentation PR #[13614](https://github.com/DataDog/documentation/pull/13614).

### Review checklist

- [ ] Feature or bugfix MUST have appropriate tests (unit, integration)
- [ ] Make sure each commit and the PR mention the Issue number or JIRA reference
- [ ] Add CHANGELOG entry for user facing change.

### CODE REVIEW

File: docs/rum_mobile_vitals.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+## Overview
+
+Real User Monitoring offers Mobile Vitals, a set of metrics inspired by [Android Vitals][1], that can help compute insights about your mobile application's responsiveness, stability, and resource consumption.
```
Comment (charlesbaiden) [inline docs/rum_mobile_vitals.md:3]: [Android Vitals] -> MetricKit https://developer.apple.com/documentation/metrickit

**Review by charlesbaiden — COMMENTED** [review]
LGTM, left one comment - thanks!

File: docs/rum_mobile_vitals.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+| Slow renders                   | To ensure a smooth, [jank-free][3] user experience, your application should render frames in under 60 Hz. <br /><br />  RUM tracks the application’s [display refresh rate][4] using `@view.refresh_rate_average` and `@view.refresh_rate_min` view attributes. <br /><br />  With slow rendering, you can monitor which views are taking longer than 16 ms or 60 Hz to render. <br /> **Note:** Refresh rates are normalized on a range of zero to 60 fps. For example, if your application runs at 100 fps on a device capable of rendering 120 fps, Datadog reports 50 fps in Mobile Vitals. |
```
Comment (alai97) [inline docs/rum_mobile_vitals.md:20]:
```suggestion
| Slow renders                   | To ensure a smooth, [jank-free][3] user experience, your application should render frames in under 60Hz. <br /><br />  RUM tracks the application’s [display refresh rate][4] using `@view.refresh_rate_average` and `@view.refresh_rate_min` view attributes. <br /><br />  With slow rendering, you can monitor which views are taking longer than 16ms or 60Hz to render. <br /> **Note:** Refresh rates are normalized on a range of zero to 60fps. For example, if your application runs at 100fps on a device capable of rendering 120fps, Datadog reports 50fps in Mobile Vitals. |
```

**Review by maycmlee — APPROVED** [review]

File: docs/rum_mobile_vitals.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+| Application not responding     | When the UI thread of an application is blocked for more than 5 seconds, an `Application Not Responding` ([ANR][6]) error triggers. If the application is in the foreground, the system displays a dialog modal to the user, allowing them to force quit the application. <br /><br />   RUM tracks ANR occurrences and captures the entire stack trace that blocks the main thread when it encounters an ANR.                                                                                                                                                                                                                              |
```
Comment (ncreated) [inline docs/rum_mobile_vitals.md:22]: There is no ANR concept on iOS - IMO we should skip this part from iOS doc.
```suggestion
```

**Review by ncreated — APPROVED** [review]
Looks good 👍, I only left one comment on ANRs which don't have a place in iOS. Thx @alai97!

**Pull request merged**